### DISCUSSION TITLE
Add `enabled` option

### DISCUSSION THREAD

**Discussion by m-shaka** [biomejs/biome-vscode#149] [Ideas]
I would like to add `enabled` option.
The extension is always enabled when I have a global binary of `biome`, but some projects are not using `biome` as the linter or the formatter.

**Comment by nhedger** [comment]
It's on the way! Cannot guarantee it but I'm hoping to make this a part of the next release, and if not, shortly after.

**Reply by m-shaka** [reply]
Thank you! That's a good news
